client/payment-details/index.tsx

```diff
@@ -18,6 +20,20 @@ const PaymentDetails: React.FC< PaymentDetailsProps > = ( { query } ) => {
 		transaction_type: transactionType,
 	} = query || {};
 
+	const { status_is: statusIs, type_is: typeIs } = getQuery();
+
+	if ( statusIs && typeIs ) {
+		wcpayTracks.recordEvent(
+			'wcpay_fraud_protection_order_details_link_clicked',
+			{ status: statusIs, type: typeIs }
+		);
+
+		updateQueryString( {
```

Comment:
Can we add a comment here in the newly added block, so people don't remove the block mistakenly?

Reply:
Ok, I added this about `updateQueryString`:
> // Remove the tracking queries on page load so we don't track refreshes or back button reloads.

Fixed in [498643a](https://github.com/Automattic/woocommerce-payments/pull/5820/commits/498643a79759ac936ecc33e22f2e96ae6eb7a2e0).